### PR TITLE
Pytest/parametrize test simple

### PR DESCRIPTION
**Pull request by palao**
The old `test_simple.py` has been parametrized and rewritten as `test_baseline.py` + `conftest.py`. Now, instead of one large test, it is 28 smaller tests that have been `pytest`-marked  with `ft`.

### CODE REVIEW

**Comment by thesamesam** [comment]
Try rebasing on master?

**Comment by palao** [comment]
It fails miserably, but I don't understand the error. It seems to be meson related?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -36,16 +36,10 @@ jobs:
           python -VV
           python -m site
           python -m pip install --upgrade pip
-          # setuptools needed for 3.12+ because of https://github.com/mesonbuild/meson/issues/7702.
```
Comment (thesamesam) [inline .github/workflows/ci.yml:39]: You seem to have undone the hack we were doing here before. Any particular reason or was it accidental?

I think that explains the CI failure.

File: NEWS [hunk 2]
```diff
@@ -156,6 +156,8 @@ to meson-python, thanks to Chewi!
 
 Breaking changes:
 * The minimum supported Python version is now >= Python 3.9.
+* Portage now installed with Meson and Python sdist + wheel now prepared with
```
Comment (thesamesam) [inline NEWS:159]: Drop -- merge conflict error.

File: lib/portage/tests/runTests.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
```
Comment (thesamesam) [inline lib/portage/tests/runTests.py:1]: Drop again? Looks like a merge conflict error (we don't want runTests.py now, right?)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (thesamesam) [reply .github/workflows/ci.yml:39]: I think you want to drop 9284e0df9670f4dc30e9d547d5ba33ad4c70111d from your branch.

File: lib/portage/tests/runTests.py (hunk 3 above)
Comment (thesamesam) [reply lib/portage/tests/runTests.py:1]: -> https://github.com/gentoo/portage/pull/1146#discussion_r1369429609

File: NEWS (hunk 2 above)
Comment (thesamesam) [reply NEWS:159]: -> https://github.com/gentoo/portage/pull/1146#discussion_r1369429609

File: .github/workflows/ci.yml (hunk 1 above)
Comment (palao) [reply .github/workflows/ci.yml:39]: Sorry for the delay. It was accidental, of course. No idea why it ended up in my branch...

I'll remove it and force push.

**Comment by palao** [comment]
Now it works. Sorry for the mess!
What is next?

**Review by zmedico — APPROVED** [review]
Looks great! Thanks!

I tried pytest-xdist with `pytest -m ft -n 8` and it was fast!

**Comment by thesamesam** [comment]
Thank you Zac and thank you David for this fantastic work!

**Comment by thesamesam** [comment]
`pytest -n 32 --dist=worksteal` took 65s for me!

**Pull request closed without merging**